frontend/src/entry/model/releases/EditableRelease.tsx

```diff
@@ -43,6 +43,7 @@ export default function EditableRelease({ release, isEdit, onIsEditChange }: Edi
   const [imageList, setImageList] = useState<FlattenedModelImage[]>(release.images)
   const [errorMessage, setErrorMessage] = useState('')
   const [isLoading, setIsLoading] = useState(false)
+  const [registryError, setRegistryError] = useState(false)
```

Comment:
At a glance I would assume this is a string that holds the text for an error. Consider renaming to `isRegistryError` and `setIsRegistryError` to clarify that it is a boolean.